components/blitz/src/ome/services/blitz/fire/Registry.java

```diff
@@ -62,7 +62,9 @@
      * interval     := Seconds between retries
      * client_uuid  := Uuid of the client which should be used
      * </pre>
+     * @deprecated in OMERO 5.4 uses group ID instead of group name
```

Comment:
the way it is displayed can be a bit confusing. "deprecated in OMERO 5.4".
Maybe be to have ``In`` upper case